front/app/components/PageForm/index.tsx

```diff
@@ -101,6 +107,11 @@ const PageForm = ({
           <Feedback
             successMessage={formatMessage(messages.savePageSuccessMessage)}
           />
+        </SectionField>
+        <SectionField>
+          <Outlet id="app.components.PageForm.index.top" pageId={pageId} />
```

Comment:
I meant that you should not put the Outlet inside SectionField. SectionField needs to be added in the module, wrapped around

`<NavbarTitleField pageId={pageId} />`

If we want to use this outlet in the future for another form field, both will be inside the same SectionField, which will mess up the styles. So there should be no SectionField wrapping here, just the Outlet.

Reply:
As `NavbarTitleField` sometimes returns `null`, I used `SectionField` right inside of the component to not have magins around `null`.

Reply:
Null means rendering nothing. Have you tested it?

Reply:
If margin would render (need to check, not sure about it; but padding does render), it's an extra reason to not render SectionField if there's going to be nothing inside it.

Reply:
Yes, I think it's padding. 

> Null means rendering nothing. Have you tested it?

Yes, the navbar title is not rendered on the policy form 
![image](https://user-images.githubusercontent.com/91883434/186376559-49f0849f-55a3-49ab-900e-d3560d15a129.png)